src/LinkedIn.php

```diff
@@ -659,7 +659,8 @@ protected function filterRequestOption(array &$options)
         switch ($options['format']) {
             case 'simple_xml':
                 $options['simple_xml'] = true;
-            // simple_xml is still xml. This should fall through
+                $options['headers']['Content-Type'] = 'text/xml';
+                break;
```

Comment:
Somehow I really like not using the break here. But you are right, it is more clear and it gets easier to read.